Source/WebKit/UIProcess/Cocoa/SOAuthorization/SOAuthorizationSession.mm

```diff
@@ -235,6 +235,18 @@ static bool isSameOrigin(const WebCore::ResourceRequest& request, const WebCore:
         SOAuthorizationOptionInitiatorOrigin: (NSString *)initiatorOrigin,
         SOAuthorizationOptionInitiatingAction: @(static_cast<NSInteger>(m_action))
     };
+#if PLATFORM(VISION)
```

Comment:
Any /harm/ in not conditionalizing this?

Reply:
Good question, no (because the delegate method impl is already guarded), so I removed the platform guard.

Reply:
Actually, put back in a platform guard because the delegate method is only defined on IOS platforms. But at your suggestion made it more general (IOS_FAMILY). Thanks!